### ISSUE TITLE
Teleportation does not work for WMR. Bezier line sticks to the floor

### ISSUE THREAD

**Issue by wassx** [XRTK/XRTK-Core#385]
# XRTK - Mixed Reality Toolkit Bug Report

## Describe the bug

I added all the latest xrtk packages to an empty unity 2019.1.14f1 project (UWP).

Added packages are:

```
"com.xrtk.core": "0.1.23",
 "com.xrtk.oculus": "0.1.5",
 "com.xrtk.sdk": "0.1.15",
 "com.xrtk.wmr": "0.1.7",
```

After "Mixed Reality Toolkit -> configure project" scene was setup and I cloned all relevant profiles without any changes.
For the controllerprovider I deactivated all (target none) except the WMR for editor.

When I run the scene the white line is sticky to the floor and no teleportation works.
![image](https://user-images.githubusercontent.com/3955210/66637086-adbe3100-ec12-11e9-8ed3-25f481de6b19.png)
![image](https://user-images.githubusercontent.com/3955210/66637146-c29ac480-ec12-11e9-912c-ef1a63d0ab88.png)
![image](https://user-images.githubusercontent.com/3955210/66637164-cdedf000-ec12-11e9-99b3-aadf2c6df1ca.png)

## Your Setup (please complete the following information)

- Unity Version [e.g. 2019.1.14f1]
- XRTK Version [e.g. 0.1.23]

## Target Platform (please complete the following information)

- WMR immersive

## Example Project
[LaserUnicorn.zip](https://github.com/XRTK/XRTK-Core/files/3716436/LaserUnicorn.zip)

**Comment by SimonDarksideJ** [comment]
So I recreated a project based on your steps and got a completely different result, not working, just very different.  Going to try your project next as I'm perplexed :S

I get this with just WMR now (not with any other platform)
![image](https://user-images.githubusercontent.com/1793042/66687369-5489e800-ec7a-11e9-814e-115eba3887ce.png)

**Comment by SimonDarksideJ** [comment]
Right, I like simple answers, although it took me a while to find it.
The answer to this is very simple, you disabled the WMR platform from running in the editor
![image](https://user-images.githubusercontent.com/1793042/66688442-84d38580-ec7e-11e9-8dee-6ab8def75eab.png)

When you configured the platforms, you changed the UWP controllers to only run in Windows Universal.  If you build and run your project on Windows 10, you will find your controllers there.

SO basically, the XRTK is doing what you told it to do and didn't start the Mixed Reality controllers because you told it not to in the editor :D

**Comment by StephenHodgson** [comment]
I bet this was an artifact from the breaking change we did to add the editor platform to the configuration settings

**Comment by SimonDarksideJ** [comment]
Nope, pretty sure this is expected behaviour, if you unselect editor, it doesn't run in the editor :P
